tekton/mario-bot/mario-github-comment.yaml

```diff
@@ -182,7 +183,6 @@ spec:
     - setup-comment
     taskRef:
       name: pull-request
-      bundle: gcr.io/tekton-releases/catalog/upstream/pull-request:0.1
```

Comment:
thanks for explaining this change in the commit message! i saw this line and i was like "im going to recommend explaining why this happened in the commit" AND IT'S ALREADY THERE 🎉 👍 💯  nice!!